tests/test262/interpret-results.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+module.exports = function report(results, allowed) {
+  var expected = {
+    success: [],
+    failure: [],
+    typeI: [],
+    typeII: []
+  };
+  var unexpected = {
+    success: [],
+    failure: [],
+    typeI: [],
+    typeII: [],
+    unrecognized: null
+  };
+  var totalUnexpected;
+
+  results.forEach(function(result) {
+    var isAllowed = allowed[result.name];
+    delete allowed[result.name];
+
+    if (!!result.parseFailure === result.expected) {
+      if (isAllowed) {
+        if (result.parseFailure) {
+          unexpected.failure.push(result.name);
+        } else {
+          unexpected.success.push(result.name);
+        }
+      } else {
+        if (result.parseFailure) {
+          expected.failure.push(result.name);
+        } else {
+          expected.success.push(result.name);
+        }
+      }
+    } else {
+      if (isAllowed) {
+        if (result.parseFailure) {
+          expected.typeI.push(result.name);
```

Comment:
Instead of this `typeI` / `typeII` stuff, how about names which indicate what these are? Like, `passes` and `failures` maybe? Much easier to grok what it's saying that way